extensions/file/README.md

```diff
@@ -9,7 +9,7 @@
 
 Provides a way to specify file related details such as checksum, data type and size for assets in [STAC Items](../../item-spec/item-spec.md) and [STAC Collections](../../collection-spec/collection-spec.md) implementing the [Collection Assets](../collection-assets/README.md) extension.
 
-- [Example](examples/sentinel1.json)
+- [Example](../../examples/extensions-collection/sentinel-1-example/sentinel-1-example.json)
```

Comment:
We should leave examples in the extensions if they don't go to core, because then the examples are missing once we fork for the stac-extensions org.

Reply:
Good point. Will move them back. This was started before we had our extension decisions, and I thought sar and file might make it...